ui/app/AppLayouts/Browser/popups/BrowserWalletMenu.qml

```diff
@@ -212,4 +212,7 @@ Popup {
             }
         }
     }
+    onClosed: {
+        popup.destroy();
```

Comment:
Same here

Reply:
hmm this is a `Popup` type, wdym?

Reply:
Yeah but did we create it ourselves explicitly?

Reply:
yesh, it's instantiated in BrowserLayout.qml line 427